lib/src/models/models.dart

```diff
@@ -189,6 +189,8 @@ abstract class AbstractControl<T> {
     updateValue(value);
   }
 
+  dynamic get jsonValue => _value;
```

Comment:
Adding a separate `jsonValue` method should be a good solution to the above problem.